packages/@uppy/core/src/Uppy.js

```diff
@@ -44,6 +45,8 @@ class Uppy {
 
   #postProcessors = new Set()
 
+  queue = new PQueue({ concurrency: 6, autoStart: false })
```

Comment:
This should be user configurable, no?

Reply:
If we don't want breaking changes, we probably override it per upload plugin:

```js
// inside xhr-upload
this.uppy.queue.concurrency = this.opts.limit
```